### ISSUE TITLE
If ya can lend a hand, moving from 7 -> 8. Supposedly stock db. 

### ISSUE THREAD

**Issue by o-w** [OCA/OpenUpgrade#282]
2015-04-22 19:24:53,913 1789 ERROR bitnami_openerp_migrated OpenUpgrade: Invalid value 'tree_account_reconciliation' in the table 'ir_ui_view' for the field 'type'. (1 rows).

then later in same log...

RROR bitnami_openerp_migrated openerp.sql_db: bad query: ALTER TABLE "stock_picking" ALTER COLUMN "picking_type_id" SET NOT NULL
Traceback (most recent call last):
  File "/var/tmp/openupgrade/8.0/server/openerp/sql_db.py", line 234, in execute
    res = self._obj.execute(query, params)
IntegrityError: column "picking_type_id" contains null values

2015-04-22 19:28:35,343 1789 WARNING bitnami_openerp_migrated openerp.models: WARNING: unable to set column picking_type_id of table stock_picking not null !
Try to re-run: openerp-server --update=module
If it doesn't work, update records and execute manually:
ALTER TABLE stock_picking ALTER COLUMN picking_type_id SET NOT NULL

(I ran openerp-server --update=module, then re-ran script but no go. I'll go in and update the DB manually next... )

and even further down...

similar complaint asking to manually run:
ALTER TABLE stock_warehouse ALTER COLUMN view_location_id SET NOT NULL

So basically I go in and run these SQL statements then re-run migration script? Any other tips?

**Comment by o-w** [comment]
Another similar issue posted by https://github.com/OpenUpgrade/OpenUpgrade/issues/283

Closing this one as its redundant.
